src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -355,7 +355,7 @@ public Map<String, String> onIncoming(@NotNull IChannel channel, @NotNull ByteBu
             sendLogout();
             reconnectRequestTimer = executorService.schedule(this::sendLogon, settings.getReconnectDelay(), TimeUnit.SECONDS);
             metadata.put(REJECT_REASON, "SeqNum is less than expected.");
-            if (LOGGER.isErrorEnabled()) LOGGER.error("Invalid message. SeqNum is less than expected {}: {}", serverMsgSeqNum.get(), message.toString(US_ASCII));
+            error("Invalid message. SeqNum is less than expected %d: %s", null, serverMsgSeqNum.get(), message.toString(US_ASCII));
```

Comment:
Could we execute message.toString(US_ASCII) operation here and other places if log level is enabled?